src/svg/HomeSVG.js

```diff
@@ -10,6 +11,7 @@ Animated.addWhitelistedNativeProps({
 
 const HomeSVG = ({color, size}: SVGProps) => {
   return (
+    <View testID="FeedSVG">
```

Comment:
Try to choose between using either camelCase or lisp-case for testIDs in order to maintain consistency